src/websocket.js

```diff
@@ -393,6 +393,14 @@ Strophe.Websocket.prototype = {
      * (Object) error - The websocket error.
      */
     _onError: function(error) {
+        // Workaround
+        // https://github.com/strophe/strophejs/pull/145
+        // https://code.google.com/p/chromium/issues/detail?id=372776
+        // 
+        if (!this.socket || this.socket.readyState === 0) {
+          return;
+        }
```

Comment:
don't ask me why; sometimes socket onclose is called before onerror and sometimes not that's why there is a !this.socket check